static/panes/tree.ts

```diff
@@ -22,9 +22,8 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import { MultifileFile, MultifileService, MultifileServiceState } from "../multifile-service";
-import { saveAs } from "file-saver";
-import { LineColouring } from "../line-colouring";
+import {MultifileFile, MultifileService, MultifileServiceState} from '../multifile-service';
```

Comment:
I don't remember changing this stuff...is it possible a prettier thing?

Reply:
must be

Reply:
Can't be. `requirePragma` is set in the prettier config and this file (As well as the rest of the files in the repo) lacks it

Reply:
hmm, weird, maybe matt's IDE did it?

Reply:
when I'm doing changes I don't see this either, so I guess it's true what you say @RubenRBS

Reply:
Yes, very likely it was my IDE that did it (I hit reformat in the IDE which drops redundant includes) before I found the "format using Prettier" button.